plugins/techdocs/src/plugin.ts

```diff
@@ -60,6 +49,11 @@ export const rootCatalogDocsRouteRef = createRouteRef({
 export const techdocsPlugin = createPlugin({
   id: 'techdocs',
   apis: [
+    createApiFactory({
+      api: scmIntegrationsApiRef,
+      deps: { configApi: configApiRef },
+      factory: ({ configApi }) => ScmIntegrationsApi.fromConfig(configApi),
+    }),
```

Comment:
Let's not, this is supposed to be provided by the app and not any particular plugin

Reply:
Thanks, I am stumbling a bit on the api references. Thought I got a "could not find `intgration.scmxyz` ref" error without it, but seems that might have been something else. Removing it to see if all tests still pass